assets/app/views/edit/build-config.html

```diff
@@ -0,0 +1,526 @@
+                            </div>
+                          </dl>
+                        </div>
+                      </div>
+
+
+                      <div class="col-lg-6">
+                        <div class="section">
+                          <h3>Environment Variables<span class="help action-inline">
+                            <a href>
+                            <i class="pficon pficon-help" data-toggle="tooltip" data-placement="right" aria-hidden="true" data-original-title="Environment variables are used to configure and pass information to running containers.  These environment variables will be available during your build and at runtime."></i>
+                            </a>
+                          </span></h3>
+                          <div class="form-groups">
+                            <osc-key-values
+                              entries="envVars"
+                              delimiter="="
+                              key-validator="env"
+                              delete-policy="always"
+                              key-title="Name"
+                              key-validation-tooltip="A valid environment variable name is an alphanumeric (a-z and 0-9) string beginning with a letter that may contain underscores."></osc-key-values>
+                          </div>
+                        </div>
+                        <div class="section">
+                          <h3>Triggers</h3>
+                          <dl class="dl-horizontal left">
+                            <div class="form-groups">
+                              <div class="checkbox" ng-if="sources.git" >
+                                <label>
+                                  <input type="checkbox" ng-model="pickedTriggers['webhook']"/>
+                                  Enable the webhook build trigger
+                                  <span class="help action-inline">
+                                    <a href>
+                                      <i class="pficon pficon-help" data-toggle="tooltip" data-placement="right" aria-hidden="true" data-original-title="The source repository must be configured to use the webhook to trigger a build when source is committed.">
+                                      </i>
+                                    </a>
+                                  </span>
+                                </label>
+                              </div>
+                              <div class="checkbox">
+                                <label>
+                                  <input type="checkbox" ng-model="pickedTriggers['imageChange']" ng-disabled="builderOptions.pickedType === 'None'"/>
+                                  Automatically build a new image when the builder image changes
+                                  <span class="help action-inline">
+                                    <a href>
+                                      <i class="pficon pficon-help" data-placement="right" aria-hidden="true"
```

Comment:
These tooltips should have `data-placement="left"` (or perhaps top or bottom) since they're near the right edge of the screen.